README.md

```diff
@@ -33,21 +33,38 @@ make help
 In order to develop any `apt` related items, the `python-apt` package is needed.
 The `apt` extra will require this package in general.
 
-Apt package prerequisites for this development environment on an Ubuntu system can be installed with:
+> Ubuntu 22.04 is assumed in the rest of this setup
 
+Development packages, such as those to build the Apt Python package are required:
+
+```bash
+sudo apt install libapt-pkg-dev intltool fuse-overlayfs python3.10-venv python3-dev gcc g++ make
+```
+
+The following are required to run integration tests (it is a default target for `make tests`):
 ```bash
-sudo apt install libapt-pkg-dev intltool fuse-overlayfs
+sudo snap install chisel --candidate
+sudo apt install ninja-build cmake scons qt5-qmake p7zip rpm \
+                 autoconf automake autopoint gcc git gperf help2man libtool texinfo \
+                 pkg-config
 ```
 
-On a Debian or Ubuntu system, the appropriate package can be installed by running
+Create virtual environment to develop in and activate it:
 
 ```bash
-apt source python-apt
-pip install ./python-apt_*
+python3 -m venv .venv
+source .venv/bin/activate
 ```
 
-or by downloading and installing the appropriate source tarball. On Ubuntu these are:
+Install the required Python packages
 
+```bash
+pip install .[dev]
+pip install .[docs]
```

Comment:
It's probably best to install craft-parts in editable mode so you don't have to reinstall after code changes.

Also, these can be done in one command.

```suggestion
pip install -e .[dev,docs]
```